o2a/converter/oozie_converter.py

```diff
@@ -136,11 +143,24 @@ def convert_nodes(nodes: Dict[str, ParsedActionNode]):
             p_node.tasks = tasks
             p_node.relations = relations
 
-    def add_properties_to_params(self, params: Dict[str, str]):
+    def read_configuration_properties(self):
+        """
+        Reads configuration job_properties to configuration_properties dictionary.
+        Replaces EL job_properties within.
```

Comment:
```suggestion
        Replaces EL configuration_properties within.
```

Reply:
That was a bad rename properties -> job_properties at some point of time. Resolved in 3d1455955ce0a10df49cf13602d02696f6c89318